mteb/tasks/Retrieval/eng/ClimateFEVERRetrievalFast.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import annotations
+
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+from ....abstasks.AbsTaskRetrieval import AbsTaskRetrieval
+
+
+class ClimateFEVERFast(AbsTaskRetrieval):
+    metadata = TaskMetadata(
+        name="ClimateFEVER-Fast",
+        description="CLIMATE-FEVER is a dataset adopting the FEVER methodology that consists of 1,535 real-world claims regarding climate-change. ",
```

Comment:
(general comment)

We should add a description of how it differs from the original dataset here.

Reply:
Added!